server/appointment/upload-documents/uploadDocuments.php

```diff
@@ -125,9 +125,10 @@ function uploadDocument($token, $firstName, $lastName, $emailAddress, $phoneNumb
 		$clientInformation = validateClientInformation($token, $firstName, $lastName, $emailAddress, $phoneNumber);
 		$appointmentId = $clientInformation['appointmentId'];
 		$appointmentType = $clientInformation['appointmentType'];
-
+		$siteId = $clientInformation['siteId'];
+		
 		// Upload the user's file to Azure BLOB Storage
-		$containerName = 'ty2019';
+		$containerName = getContainerName($siteId);
```

Comment:
Oh nice! I forgot there were top-level "containers" in Azure BLOB that we can permission access to. This is brilliant--will avoid us having to provision a new storage account for every site.